README.md

```diff
@@ -4,23 +4,8 @@ Module to work with an IBM Cloud Object Storage instance. If the `provision` fla
 
 **Note:** This module follows the Terraform conventions regarding how provider configuration is defined within the Terraform template and passed into the module - https://www.terraform.io/docs/language/modules/develop/providers.html. The default provider configuration flows through to the module. If different configuration is required for a module, it can be explicitly passed in the `providers` block of the module - https://www.terraform.io/docs/language/modules/develop/providers.html#passing-providers-explicitly.
 
-## Software dependencies
 
-The module depends on the following software components:
-
-### Command-line tools
-
-- terraform - v0.15
-
-### Terraform providers
-
-- IBM Cloud provider >= 1.18
-
-## Module dependencies
-
-- Resource group - github.com/terraform-ibm-modules/terraform-ibm-toolkit-resource-group
-
-## Example usage
+## Usage
```

Comment:
Could we align the usage section with general guidance of using actual values (albeit dummy)